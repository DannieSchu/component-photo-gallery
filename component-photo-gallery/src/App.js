import React, { Component } from 'react';
import images from './data';
import Header from './Organisms/Header';
import ImageList from './Organisms/ImageList';
// import ImageItem from './Molecules/ImageItem';
import './App.css';

export default class App extends Component {
  render() {
    return (
      <div className="App">
        <Header />
        <ImageList images = {images} />
      </div>
    )
  }
}