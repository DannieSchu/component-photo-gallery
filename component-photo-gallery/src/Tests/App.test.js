import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';
import renderer from 'react-test-renderer';


// test('renders learn react link', () => {
//   const { getByText } = render(<App />);
//   const linkElement = getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });

test('renders App.js correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });