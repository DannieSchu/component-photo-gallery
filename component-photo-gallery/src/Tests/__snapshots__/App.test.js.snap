// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
<div
  className="App"
>
  <h1>
    Header here
  </h1>
  <div>
    <p>
      There are 
      20
       images in this array. 
    </p>
    <p>
      Here's the first: 
    </p>
    <img
      alt="unicorn"
      src="http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg"
    />
  </div>
</div>
`;
